diff --git a/node_modules/expo-wallpaper-manager/android/.project b/node_modules/expo-wallpaper-manager/android/.project
new file mode 100644
index 0000000..6cf6749
--- /dev/null
+++ b/node_modules/expo-wallpaper-manager/android/.project
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>expo-wallpaper-manager</name>
+	<comment>Project expo-wallpaper-manager created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1771188412923</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
diff --git a/node_modules/expo-wallpaper-manager/expo-module.config.json b/node_modules/expo-wallpaper-manager/expo-module.config.json
index 49b9e70..3329805 100644
--- a/node_modules/expo-wallpaper-manager/expo-module.config.json
+++ b/node_modules/expo-wallpaper-manager/expo-module.config.json
@@ -1,6 +1,6 @@
 {
   "platforms": ["android"],
   "android": {
-    "modules": ["expo.modules.wallpaper.ExpoWallpaperModule"]
+    "modules": ["expo.modules.wallpapermanager.ExpoWallpaperManagerModule"]
   }
 }
